src/Examples/Advanced/CustomProperties/custom_physical_property_packages.ipynb

```diff
@@ -0,0 +1,1329 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def unfix_state(blk, flags, outlvl):\n",
+    "    init_log = idaeslog.getInitLogger(blk.name, outlvl, tag=\"properties\")\n",
+    "\n",
+    "    # Reactivate sum of mole fractions constraint\n",
+    "    for k in blk.keys():\n",
+    "        if blk[k].config.defined_state is False:\n",
+    "            blk[k].mole_fraction_constraint.activate()\n",
+    "\n",
+    "    if flags is not None:\n",
+    "        # Unfix state variables\n",
+    "        revert_state_vars(blk, flags)\n",
+    "\n",
+    "    init_log.info_high(\"State Released.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating the StateBlock and StateBlockData classes\n",
+    "\n",
+    "We have now defined almost all the of methods we will need to construct the actual State Block classes, and those that remain will be built directly as part of the classes. To begin, we need to declare the class which will be used to construct the `IndexedStateBlock`, and that contains the methods associated with initializing the State Block. Whilst this might seem a little counter-intuitive (defining initialization before we define the actual model), it is necessary to do it this way as we need to have defined this class before we can define the `StateBlockData` class (for reasons that you will see later).\n",
+    "\n",
+    "### The StateBlock class\n",
+    "\n",
+    "To begin, we declare a new class which inherits from the IDAES `StateBlock` base class, which we will call `_HDAStateBlock` here. Note that we need to be careful with naming here; later we will be automatically generating a class names `HDAStateBlock` which will be the actual state block that we use for our model (and incidentally the one we created a pointer to back in the Physical Parameter Block class). Thus, we add an underscore to the beginning of this class name to distinguish it from the class that will be generated later, and to indicate that this is for internal use only by the property package.\n",
+    "\n",
+    "Next, the `_HDAStateBlock` class needs to have two methods added to it; `initialize` and `release_state` which are used to run the initialization routine for the State Block. Each of these have a set of arguments which must part of the method definition (you can add more if you have a need for them, but the pre-defined set are a minimum).\n",
+    "\n",
+    "#### The `initialize` method\n",
```

Comment:
The `initialize` method just needs a quick update to (1) match the new usage of `get_solver()` and (2) set default arguments to `None` rather than `{}`